Makefile

```diff
@@ -1,83 +1,41 @@
-#### Terraform | Scaleway provider
-init-scw:
-	terraform -chdir=deploy/providers/scaleway init
-
-plan-scw:
-	@make init-scw
-	terraform -chdir=deploy/providers/scaleway plan	
-	
-deploy-scw:
-	@make init-scw
-	terraform -chdir=deploy/providers/scaleway apply
-
-destroy-scw:
-	terraform -chdir=deploy/providers/scaleway destroy
-
-### Docker usage
+run: up
+	@make create-seeds
 
-# Docker images commands
+up:
```

Comment:
Can you backport this PR on the bump ? 
https://github.com/OpenSourcePolitics/decidim-app/pull/554/files